### PR TITLE
Update/store static info token list

### PR DESCRIPTION
**Pull request by dospore**
# Motivation
- Reuse the known assets from the bridge states in the token lists and quoteToken

# Changes
- reference quote token from token mapping
- construct poolTokenList from poolList
- construct token mapping from tokenList

### CODE REVIEW

File: libs/constants/poolLists.ts [hunk 1]
```diff
@@ -232,7 +213,38 @@ export const poolList: Record<AvailableNetwork, StaticPoolInfo[]> = {
...
+                        ...poolInfo.shortToken
+                    },
+                    [poolInfo.longToken.address]: {
+                        ...poolInfo.longToken
+                    },
+                }))
+            )
+        })
+    )
+);
```
Comment (ScaredIbis) [inline libs/constants/poolLists.ts:250]: gnarly

File: libs/constants/tokenList.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                ...tokenList[key as AvailableNetwork].map((token) => ({
+                    [token.symbol]: {
+                        ...token
+                    },
+                }))
+            )
+        })
+    )
+);
+console.log(tokenMap)
```
Comment (ScaredIbis) [inline libs/constants/tokenList.ts:57]: boi

**Comment by ScaredIbis** [comment]
image

Looks like some the static config for the long tokens are missing the `L`

File: libs/constants/poolLists.ts (hunk 1 above)
Comment (dospore) [reply libs/constants/poolLists.ts]: hekkas JS

**Review by ScaredIbis — APPROVED** [review]

**Pull request merged**